Formula/s/stellar-cli.rb

```diff
@@ -0,0 +1,32 @@
+class StellarCli < Formula
+  desc "Stellar command-line tool for interacting with the Stellar network"
+  homepage "https://developers.stellar.org"
+  url "https://github.com/stellar/stellar-cli.git",
+      tag:      "v21.3.0",
+      revision: "11746937e813346272faf68e70cbdbc1b0df780f"
+  license "Apache-2.0"
+  head "https://github.com/stellar/stellar-cli.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "rust" => :build
+  depends_on "openssl@3"
+
+  on_linux do
+    depends_on "pkg-config" => :build
+  end
```

Comment:
```suggestion
  depends_on "pkg-config" => :build
  depends_on "rust" => :build
  depends_on "openssl@3"
```
`rust` already depends on `pkg-config` anyway (and the reason why it may seem like it's not a dependency on macOS is that it's always on `PATH` for macOS builds).